modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache

```diff
@@ -252,6 +252,26 @@ public class ApiClient {
     return scheme + "://" + host + (port == -1 ? "" : ":" + port) + basePath;
   }
 
+  /**
+   * Set a custom schema for the target service, for example 'https'.
+   *
+   * @param schema The schema of the target service
+   * @return This object.
+   */
+  public ApiClient setSchema(String schema){
+    this.schema = schema;
```

Comment:
I think it should be `scheme`, not `schema`

Reply:
Fixed the typo